src/test/unit/web_interface/test_plugin_routes.py

```diff
@@ -0,0 +1,81 @@
+from flask import Flask
+from flask_restful import Api
+from itertools import chain
+import os
```

Comment:
standart imports os, unittest and itertools should come before flask, flask_restful